docs/docs/guides/integrate/login/login-users.mdx

```diff
@@ -0,0 +1,240 @@
+
+Overall, authentication with SAML provides a secure and standardized method for enabling single sign-on (SSO) and federated identity management across different applications and systems.
+It allows users to access multiple services with a single set of credentials, streamlining the authentication process while maintaining strong security measures.
+
+Note that SAML might not be suitable for mobile applications.
+In case you want to integrate a mobile application, use OpenID Connect or our Session API.
+
+There are more [differences between SAML and OIDC](https://zitadel.com/blog/saml-vs-oidc) that you might want to consider.
+
+### ZITADEL's Session API
+
+ZITADEL's [Session API](/docs/apis/resources/session_service) provides developers with a straightforward method to manage user sessions within their applications.
+The Session API is not an industry-standard and can be used instead of OpenID Connect or SAML to authenticate users by [building your own custom login user interface](/docs/guides/integrate/login-ui).
+
+#### Tokens in the Session API
+
+The session API will return a session token that can be used to authenticate users from your application.
+This token should not be confused with am access or id tokens in opaque or JWT form that is issued during OpenID connect flows.
+
+:::info Token exchange
+Token exchange between Session API and OIDC / SAML tokens is not possible at this moment.
+:::
+
+#### Key features of the Session API
+
+These are some key features of the API:
+
+1. **Session Management**: The Session API allows you to create, retrieve, update, and delete sessions for users within your application. Sessions represent a user's active login session, providing temporary access to your application's resources.
+
+2. **Creating Sessions**: With the Session API, you can initiate a new session for a user after they successfully authenticate through ZITADEL. This typically involves generating a session token or ID, which is then associated with the user's identity and used to track their session activity.
+
+3. **Retrieving Sessions**: You can retrieve information about existing sessions using the Session API, such as session ID, user ID, creation time, and expiration time. This allows you to monitor active user sessions and retrieve session details as needed for auditing or troubleshooting purposes.
+
+4. **Updating Sessions**: The Session API enables you to update session attributes or extend session lifetimes based on specific application requirements. For example, you might want to refresh a session token periodically to prevent it from expiring prematurely or update session metadata to reflect changes in user permissions or settings.
```

Comment:
The main purpose of the update session is, that you can constantly add the verification checks you did. username check, password check, mfa- check. 
Tried to include, maybe you have a better way to write that.

```suggestion
4. **Updating Sessions**: The Session API enables you to update session attributes or extend session lifetimes based on specific application requirements. Each authentication step, such as username / password check or multi-factor verification, will result in an update session. Also you might want to refresh a session token periodically to prevent it from expiring prematurely or update session metadata to reflect changes in user permissions or settings.
```